src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -32,6 +83,17 @@ export class SurgingTempest extends Analyser {
 				}),
 			],
 		}))
+		this.suggestions.add(new TieredSuggestion({
+			content: <Trans id="war.stormseye.suggestions.overwrite.content">
```

Comment:
This key doesn't match the module handle or what the string is being used for.
```suggestion
			content: <Trans id="war.surgingtempest.suggestions.early.content">
```